docs/sites.yml

```diff
@@ -6796,8 +6796,8 @@
   description: >-
     Recipes and videos with the best of Ecuadorian cuisine.
     Collectable recipes from Diario El Universo.
-  main_url: https://recetas-eu.netlify.com/
-  url: https://recetas-eu.netlify.com/
+  main_url: https://recetas-eu.now.sh/
+  url: https://recetas-eu.now.sh/
```

Comment:
I don't understand why you changed this? Are you involved with this project? Both links seem to work. Seems out of scope of you adding the Quejamear showcase

Reply:
I understand that it is out of scope, but I wanted to take advantage of the PR, to update one of my previous sites.